docs/index.rst

```diff
@@ -37,7 +40,7 @@ Tablib is an :ref:`MIT Licensed <mit>` format-agnostic tabular dataset library,
    - {Age: 21, First Name: Bessie, Last Name: Monke}
 
    >>> data.xlsx
-   <censored binary data>
```

Comment:
Censored: _Having had objectionable content removed_; _expurgated by a censor: as, censored war news_. [ref](https://www.wordnik.com/words/censored)
Redacted: also covers "censored", but *additionally* has the meaning: _To make ready for publication; edit or revise_. [ref](https://www.wordnik.com/words/redact), which _censored_ does not have.

The content at issue here is not objectionable or illegal, and you aren't acting in the role of _censor_. The content is irrelevant, and you're acting as the _editor_ presenting the relevant information in a readable form. 

Censor:

> A person authorized to examine books, films, or other material and to remove or suppress what is considered morally, politically, or otherwise objectionable.
n. An official, as in the armed forces, who examines personal mail and official dispatches to remove information considered secret or a risk to security.
n. One that condemns or censures.
[ref](https://www.wordnik.com/words/censor)